src/reducers/assetsReducer.js

```diff
@@ -37,12 +43,10 @@ export type AssetsReducerAction = {
 
 export const initialState = {
   supportedAssets: { ethereum: [] },
+  popularAssets: { ethereum: [] },
 };
 
-export default function assetsReducer(
-  state: AssetsReducerState = initialState,
-  action: AssetsReducerAction,
-) {
+export default function assetsReducer(state: AssetsReducerState = initialState, action: AssetsReducerAction) {
   switch (action.type) {
     case SET_SUPPORTED_ASSETS:
```

Comment:
Just a thought, should we have a check right at the start to check if `action` is null or not, otherwise cases like `action.type` and `action.payload` could crash the app.

You could also change it to `action?.type` but I figure checking right at the start is the easiest way

Reply:
I think not changes need in this. I check every scenario app was not crash